src/test/java/org/example/numerals/RomanNumsTest.java

```diff
@@ -0,0 +1,19 @@
+package org.example.numerals;
+
+import org.junit.jupiter.api.DisplayName;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class RomanNumsTest {
+
+
+    @DisplayName("Convert the Roman Numerals to Numbers")
+    @org.junit.jupiter.api.Test
```

Comment:
could be better if they were separated to understand the three configurations were inspected, these are:
I can be placed before V (5) and X (10) to make 4 and 9. 
X can be placed before L (50) and C (100) to make 40 and 90. 
C can be placed before D (500) and M (1000) to make 400 and 900.